packages/vitest/rollup.config.js

```diff
@@ -57,6 +57,8 @@ const external = [
   'node:worker_threads',
   'node:fs',
   'rollup',
+  'rollup/parseAst',
+  'acorn',
```

Comment:
This should not be needed since we shouldn't import it directly anywhere. If it's still inlined, then we need to investigate it.

Reply:
Yeah, I'll look into that.

Reply:
I figured out why this. Because the `mlly` package is place in the devDependencies, so external missing `mlly`

Reply:
Good catch! The `shouldExternalize` and `isValidNodeImport` have very much the same logic, and it looks like we can just remove `isValidNodeImport`, which should be a big performance improvement!

Reply:
Hm, well, no, we can't really remove it, we still need to check if it's valid. But we can probably just inline this one function

Reply:
Okay, Yes, I tried to remove `isValidNodeImport`, but very test cases failed. Now I inline this one function and remove the logic we don't need